code/perception/src/lidar_distance.py

```
@@ -390,6 +392,30 @@ def generate_bounding_boxes(points_with_labels):
+        tuple: (ShapelyPolygon, Shape2DPolygon, Entity)
+    """
+    # Schritt 1: Shapely-Polygon aus Bounding Box
+    x_min, x_max, y_min, y_max = bbox
+    coordinates = [
+        (x_min, y_min),
+        (x_max, y_min),
+        (x_max, y_max),
+        (x_min, y_max),
+        (x_min, y_min),  # Schließe das Polygon
+    ]
+    shapely_polygon = ShapelyPolygon(coordinates)
```

Ruff (0.8.2)

Local variable `shapely_polygon` is assigned to but never used

Remove assignment to unused variable `shapely_polygon`

(F841)